src/playground-preview.ts

```diff
@@ -160,10 +160,21 @@ export class PlaygroundPreview extends LitElement {
         <mwc-icon-button
           id="reload-button"
           part="preview-reload-button"
-          icon="refresh"
           ?disabled=${!this.src}
           @click=${this._onReloadClick}
-        ></mwc-icon-button>
+        >
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 24 24"
+            fill="currentcolor"
+            width="18px"
+            height="18px"
+          >
+            <path
```

Comment:
I'd love to see a comment with the a link to the source of the path

Reply:
Added comment.